common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -59,6 +58,11 @@
     }
 
 
+    // ************* ANALYTICS *************
+
+    // Ophan pageview ID and browser ID are needed by Google Analytics, which runs sync at the bottom of the page
+    @InlineJs(ophanConfig().body, "ophanConfig.js")
```

Comment:
I see, I thought "non blocking" meant async, but on closer inspection it actually means "stick it at the bottom of the `body`". In that case I'll move it there. Thanks.